### PR TITLE
add grey block in stage page

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

**Review by NEL45 — DISMISSED** [review]

File: src/Controller/StageController.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Controller;
+
+class StageController extends AbstractController
+{
+    public function stage()
```
Comment (LucyMerlier) [inline src/Controller/StageController.php]: no return typehint?

File: src/View/User/Includes/_stage_grey_block.html.twig [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <div class="stage-grey-block-title-text">
+            <h2 class="stage-grey-block-title">
+                DETAILS
+            </h2>
+            <p class="stage-grey-block-text">
+                Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
+            </p> 
+        </div>    
+        <div class="btn-insc">
+            <button class="btn-inscription" type="button">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_stage_grey_block.html.twig]: "btn" => avoid abbreviations

File: src/View/User/Includes/_stage_grey_block.html.twig [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{% block stage_grey_block %}
+    <div class="stage-grey-block-section col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+        <div class="stage-grey-block-title-text">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_stage_grey_block.html.twig]: long class name => "grey-block-content"?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Loic-Code — DISMISSED** [review]

File: src/View/User/Includes/_stage_grey_block.html.twig (hunk 3 above)
Comment (LucyMerlier) [reply src/View/User/Includes/_stage_grey_block.html.twig]: "stage-grey-block-title-text" => too long

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Wowlfy — DISMISSED** [review]
parfait

File: src/View/User/Includes/_stage_grey_block.html.twig [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <div class="grey-block">
+            <h2 class="grey-block-title">
+                DETAILS
+            </h2>
+            <p class="grey-block-text">
+                Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
+            </p> 
+        </div>    
+        <div class="button-insc">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_stage_grey_block.html.twig]: don't use abbreviations

File: src/View/User/Includes/_stage_grey_block.html.twig [hunk 5]
```diff
@@ -0,0 +1,33 @@
+{% block stage_grey_block %}
+    <div class="grey-block-content col-xl-3 offset-xl-8 col-sm-10 offset-sm-1">
+        <div class="grey-block">
```
Comment (LucyMerlier) [inline src/View/User/Includes/_stage_grey_block.html.twig]: interverting these two class names would be clearer

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by Loic-Code — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**